api/error.go

```diff
@@ -51,3 +52,10 @@ func AlertNotFound(name string) error {
 		key:        name,
 	}
 }
+
+func FilterAlertNotFound(name string) error {
```

Comment:
In lieu with my suggestion around treating "not found" errors in commands differently. A helper like this could be useful. Similar to helpers in `os` or `io` etc.

```go
func IsFilterAlertNotFound(err error) bool {
  return errors.Is(err, EntityNotFound)
}
```

Reply:
I could not match it on a struct, however errors.As(err, &EntityNotFound{}) works fine.

However, I can't find a place where it would be useful at the moment.

Already receiving:

```
Could not fetch filter alert: filter-alert "foobar" not found
```